server/consumer.go

```diff
@@ -5055,7 +5060,7 @@ func (o *consumer) stopWithFlags(dflag, sdflag, doSignal, advisory bool) error {
 		}
 		if o.isPullMode() {
 			// Release any pending.
-			o.releaseAnyPendingRequests()
+			o.releaseAnyPendingRequests(peerRemoval)
```

Comment:
why not send a leader changed here?

Reply:
This should be handled during step-down but we will double check.